example-projects/example-project/pom.xml

```diff
@@ -122,11 +122,6 @@
                 </executions>
                 <dependencies>
                     <!-- The antenna mojo requires the kbase to fetch the license information -->
```

Comment:
If the `spdx-license-knowledge-base` dependency is removed, then this comment should be removed too, or?